rosdep/python.yaml

```diff
@@ -2564,19 +2564,15 @@ python-mock:
     xenial_python3: [python3-mock]
     yakkety: [python-mock]
     zesty: [python-mock]
-python-more-itertools-pip:
-  arch:
-    pip:
-      packages: [more-itertools]
-  debian:
-    pip:
-      packages: [more-itertools]
-  fedora:
-    pip:
-      packages: [more-itertools]
+python-more-itertools:
+  arch: [python-more-itertools]
+  debian: [python-more-itertools]
+  fedora: [python-more-itertools]
   ubuntu:
-    pip:
-      packages: [more-itertools]
+    bionic: [python-more-itertools]
```

Comment:
This should be a wildcard rule to work for focal and other newer platforms.

Reply:
But focal and newer do not use python2 anymore. So those would never be used?

Reply:
Based on the previous comment https://github.com/ros/rosdistro/pull/28473#discussion_r588783207 this seems to be fine to me.